pkg/manifests/manifests_test.go

```diff
@@ -1765,6 +1763,19 @@ func TestPrometheusUserWorkloadConfiguration(t *testing.T) {
 	if !reflect.DeepEqual(*f.config.UserWorkloadConfiguration.Prometheus.Resources, p.Spec.Resources) {
 		t.Fatal("Prometheus UWM resources not configured correctly")
 	}
+
+	for _, container := range p.Spec.Containers {
+		switch container.Name {
```

Comment:
nit: maybe we can raise in `default:` so we have a safe guard.

Reply:
changed a bit the test to make it more explicit indeed.